tests/integration/ethernet_mtu_test.py

```diff
@@ -155,3 +156,14 @@ def test_set_mtu_on_two_vlans_with_a_shared_base(eth1_up):
         libnmstate.apply(desired_state)
 
         assertlib.assert_state(desired_state)
+
+
+@ip_monitor_assert_stable_link_up('eth1')
+def test_change_mtu_with_stable_link_up():
+    desired_state = statelib.show_only(('eth1',))
```

Comment:
The ‘eth1_up’ fixture has not been used here, therefore, ‘eth1’ may not even be UP at this point.
Please use the fixture.

Reply:
Oops, fixes. Thanks!